assignments/callbacks.js

```diff
@@ -24,28 +24,56 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(items.length);
 }
 
+getLength(items, function(length) {
+  console.log("LENGTH:", length);
+});
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(`LAST ITEM: ${items[3]}`);
 }
 
+last(items, function(last) {
+  console.log(last);
+});
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y);
 }
 
+sumNums(5, 5, function(result) {
+  console.log(`RESULT ${result}`);
+});
+
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x * y);
 }
 
+multiplyNums(100, 100, function(result) {
+  console.log(`RESULT: ${result}`);
+});
+
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  for (let i = 0; i < list.length; i++) {
+    if (list.includes(item)) {
+      return true;
+    } else {
+      return false;
+    }
+  }
 }
 
```

Comment:
You're not passing true and false to a cb.
```suggestion
function contains(item, list, cb) {
  for (let i = 0; i < list.length; i++) {
    if (list.includes(item)) {
      return cb(true);
    } else {
      return cb(false);
    }
  }
}
console.log(contains("Notebook", items, function(answer) {
if (answer === true) {
return true
} else {
return false
}
}));
```